less/variables.less

```diff
@@ -11,4 +11,7 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+//Mobile
+@mobile: ~"max-width: 500px";
```

Comment:
If you include the parenthesis in your escape string, you don't need to add it everytime you use the variable.  Up to you of course though.